reference/api/index.rst

```diff
@@ -0,0 +1,180 @@
+Web API
+=======
+
+.. toctree::
+
+    rest
+    jobs
+
+How it's structured
+-------------------
+
+There is actually two APIs. A REST API with basic CRUD operations, and a jobs API which can launch heavy processes asynchronously.
+The current version of the API is v1. All URI have to request explicitly this version.
+
+.. code-block:: bash
+
+    GET https://demo.akeneo.com/api/rest/v1
```

Comment:
- API version doesn't follow the principle of shrinking perimeter of the URL.   
https://demo.akeneo.com/api/v1/rest/ or                                         
https://demo.akeneo.com/api_v1/rest                                                        
Unless we can have different version for the API. In this case, the sentence    
"The current version of the API" should be "Each API is versioned independently,
and currently both are at v1".